sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -338,21 +327,17 @@ class CertificateClient(AsyncKeyVaultClientBase):
         contain the key as well as x509 certificates. This operation requires
         the certificates/import permission.
 
-        :param name: The name of the certificate.
-        :type name: str
-        :param certificate_bytes: Bytes of the ertificate object to import.
-        This certificate needs to contain the private key.
-        :type certificate_bytes: str
-        :param password: If the private key in base64EncodedCertificate is
+        :param str name: The name of the certificate.
+        :param bytes certificate_bytes: Bytes of the ertificate object to import.
+            This certificate needs to contain the private key.
+        :param str password: If the private key in base64EncodedCertificate is
          encrypted, the password used for encryption.
-        :type password: str
         :param policy: The management policy for the certificate.
         :type policy:
          ~azure.keyvault.certificates.models.CertificatePolicy
-        :param enabled: Determines whether the object is enabled.
-        :type enabled: bool
+        :param bool enabled: Determines whether the object is enabled.
         :param tags: Application specific metadata in the form of key-value
-         pairs.
+            pairs.
```

Comment:
Looks like this would fit on the prior line.

Reply:
moved to previous line, thanks!